src/models/generator_models/avr_models.jl

```diff
@@ -300,3 +300,241 @@ function mdl_avr_ode!(
+    Vf, dVex_dt = low_pass_nonwindup(y_ll,Vex,1.0,Te,L3,L4)  
+
+    #Compute 3 States AVR ODE:
+    output_ode[local_ix[1]] = dVll_dt 
+    output_ode[local_ix[2]] = dVex_dt 
+    output_ode[local_ix[3]] = doel_dt 
+   
+    #Update inner_vars
+    inner_vars[Vf_var] = Vf  #  field voltage 
+    return
+end
+
+function mdl_avr_ode!(
+    device_states::AbstractArray,
+    output_ode::AbstractArray,
+    inner_vars::AbstractArray,
+    dynamic_device::DynamicWrapper{PSY.DynamicGenerator{M, S, PSY.EXAC1, TG, P}},
+) where {M <: PSY.Machine, S <: PSY.Shaft, TG <: PSY.TurbineGov, P <: PSY.PSS}
+
+    #Obtain references
+    V_ref = get_V_ref(dynamic_device)
+
+    #Obtain avr
+    avr = PSY.get_avr(dynamic_device)
+
+    #Obtain indices for component w/r to device
+    local_ix = get_local_state_ix(dynamic_device, typeof(avr))
+
+    #Define inner states for component
+    internal_states = @view device_states[local_ix]
+    Vm = internal_states[1]
+    Vr1 = internal_states[2]
+    Vr2 = internal_states[3]
+    Ve = internal_states[4]
+    Vr3 = internal_states[5]
+
+    #Define external states for device
+    V_th = sqrt(inner_vars[VR_gen_var]^2 + inner_vars[VI_gen_var]^2) # machine's terminal voltage
+    Vs = inner_vars[V_pss_var] # PSS output 
+    Xad_Ifd = inner_vars[Xad_Ifd_var] # machine's field current in exciter base (for the available generator models)
+
+    #Get parameters
+    Tr = PSY.get_Tr(avr)
+    inv_Tr = Tr < eps() ? 1.0 : 1.0 / Tr
+    Tb = PSY.get_Tb(avr)
+    Tc = PSY.get_Tc(avr)
+    Ka = PSY.get_Ka(avr)
+    Ta = PSY.get_Ta(avr)
+    Vr_min, Vr_max = PSY.get_Vr_lim(avr)
+    Te = PSY.get_Te(avr)
+    Kf = PSY.get_Kf(avr)
+    Tf = PSY.get_Tf(avr)
+    Kc = PSY.get_Kc(avr)
+    Kd = PSY.get_Kd(avr)
+    Ke = PSY.get_Ke(avr)
+
+    #Obtain saturation
+    Se = saturation_function(avr, Ve)
+
+    #Compute auxiliary parameters
+    I_N = Kc * Xad_Ifd / Ve
+    V_FE = Kd * Xad_Ifd + Ke * Ve + Se * Ve
+    Vf = Ve * rectifier_function(I_N)
+
+    # Compute block derivatives
+    # ###########################################################################
+    # y, dx/dt = low_pass(u, y, K, T)                                           #
+    # y_sat, 1/T * dy/dt_scaled = low_pass_nonwindup(u, y, K, T, y_min, y_max)  #
+    # y, dx/dt = high_pass(u, x, K, T)                                          #
+    # y, dx/dt = lead_lag(u,x,K,T1,T2) where x is the state                     #
+    # y = clamp(u,min,max)                                                      #
+    # ###########################################################################
+    
+    _, dVm_dt = low_pass(V_th, Vm, 1.0, 1.0 / inv_Tr)
+    V_F, dVr3_dt = high_pass(V_FE, Vr3, Kf, Tf)
+    V_in = V_ref + Vs - Vm - V_F
+    y_ll, dVr1_dt = lead_lag(V_in, Vr1, 1.0, Tc, Tb)
+    y_Vr, dVr2_dt = low_pass_nonwindup(y_ll, Vr2, Ka, Ta, Vr_min, Vr_max)
+    _, dVe_dt = integrator_nonwindup(y_Vr - V_FE, Ve, 1.0, Te, 0.0, 99999999.0)
```

Comment:
```suggestion
    _, dVe_dt = integrator_nonwindup(y_Vr - V_FE, Ve, 1.0, Te, 0.0, Inf)
```

it is better to use Inf here to make sure that the clamp is never activated